### PR TITLE
IHO S5b corrections

### PR DESCRIPTION
**Pull request by anermina**
- I added a Foreword section, since there is some additional text on the front page.
- I'm not sure if all document attributes are correct. Double checking requested in https://github.com/metanorma/mn-samples-iho/issues/30#issuecomment-630470509.

### CODE REVIEW

File: sources/s5b/sections/01-basic_subjects.adoc [hunk 1]
```diff
@@ -8,7 +8,8 @@
 |===
 | Topic/Element | Content | Learning outcomes
 
-| B1.1 Linear Algebra _(B)_
+| B1.1 Linear Algebra +
+_(B)_
```
Comment (ronaldtse) [inline sources/s5b/sections/01-basic_subjects.adoc]: @anermina let's not encode these non-semantic elements. For Metanorma, it is not important whether the `_(B)_` occurs on the same line or not. Thanks.

**Review by ronaldtse — CHANGES_REQUESTED** [review]

**Comment by ronaldtse** [comment]
@anermina we don't need to insert line-breaks 👍

File: sources/s5b/sections/01-basic_subjects.adoc (hunk 1 above)
Comment (anermina) [reply sources/s5b/sections/01-basic_subjects.adoc]: Got it! Changes are applied in both IHO S5a and IHO S5b.

**Comment by ronaldtse** [comment]
Excellent -- thanks @anermina !

**Pull request merged**